tests/baker_recipes/accounts.py

```diff
@@ -0,0 +1,8 @@
+from model_bakery.random_gen import gen_email
+from model_bakery.recipe import Recipe
+
+# Recipes for the `accounts.User` objects
+
+user = Recipe("accounts.User", email=gen_email)
+
+__all__ = ["user"]
```

Comment:
Insert extra line before `__all__`.
```suggestion

__all__ = ["user"]
```